src/blueprint-api.js

```diff
@@ -0,0 +1,555 @@
+/*
+ * decaffeinate suggestions:
+ * DS101: Remove unnecessary use of Array.from
+ * DS102: Remove unnecessary code created because of implicit returns
+ * DS205: Consider reworking code to avoid use of IIFEs
+ * DS207: Consider shorter variations of null checks
+ * Full docs: https://github.com/decaffeinate/decaffeinate/blob/master/docs/suggestions.md
+ */
+// Current Application AST version
+//
+// Beware! Since November 2015 this version number isn't applied automatically
+// to any Application AST which is created by metamorphoses. When transforming
+// API Blueprint AST, certain older ASTs get fixed to 18. See
+// api-blueprint-adapter.coffee for details.
+const CURRENT_AST_VERSION = 19;
+
+const fillProps = (object, props, defaults) =>
+  (() => {
+    const result = [];
+    for (let key in defaults) {
+      result.push(object[key] = props[key] != null ? props[key] : defaults[key]);
+    }
+    return result;
+  })()
```

Comment:
Why is this wrapped in a closure and then called immediately? Can we make `fillProps` an ordinary function?